### ISSUE TITLE
envoy 1.20: http\codec_client.cc flood console warnings

### ISSUE THREAD

**Issue by pgeler** [envoyproxy/envoy#18643]
Envoy 1.20 release start producing large amount of log messages:
```
...[1][warning][client] [external/envoy/source/common/http/codec_client.cc:122] [C2047] Connection is closed by self during connecting.
```
reason is following code:
https://github.com/envoyproxy/envoy/blob/96701cb24611b0f3aac1cc0dd8bf8589fbdf8e9e/source/common/http/codec_client.cc#L120-L123

It seems the messaging already removed from main branch, so it would be nice remove them in release as well

**Comment by ankatare** [comment]
i would like to take this

**Comment by ankatare** [comment]
@mathetake Thanks, Will get in touch with you for any help/doubts.

**Comment by mathetake** [comment]
no it is not. you are viewing v1.19 codes but this is about 1.20, and the target code is here: https://github.com/envoyproxy/envoy/blob/v1.20.0/source/common/http/codec_client.cc#L121-L122

Could you just read the  issue description again?

**Comment by mathetake** [comment]
Found that this still exists in main as opposed to what @pgeler said above, so I opened the PR against main: https://github.com/envoyproxy/envoy/pull/18699

**Comment by ankatare** [comment]
> Found that this still exists in main as opposed to what @pgeler said above, so I opened the PR against main: #18699

Hi @mathetake Yeah, actually confusion created in issue description itself see below
`"It seems the messaging already removed from main branch, so it would be nice remove them in release as well"`
Thta's what i was thinking this code already in mail so i looked back to previous release. :)

**Comment by ankatare** [comment]
> Found that this still exists in main as opposed to what @pgeler said above, so I opened the PR against main: #18699

Thanks .. will backport it now.

**Comment by mathetake** [comment]
This has been merged in release/v1.20.
